package.json

```diff
@@ -11,6 +11,7 @@
         "@protobuf-ts/plugin": "^2.0.4",
         "@protobuf-ts/runtime": "^2.0.4",
         "http-server": "13.0.1",
+        "npm": "^8.3.1",
         "sass": "^1.39.0",
         "typescript": "4.3.5"
     }
```

Comment:
It looks like you have some local workspace changes here (and in package-lock.json) that shouldn't be committed.

Reply:
Those files have been reverted back now.